### ISSUE TITLE
Unable to connect to yanbf.api.hansol.ca:443

### ISSUE THREAD

**Issue by DanLabute** [YANBForwarder/YANBF#44]
Hi there,

I'm running into an issue when trying to generate forwarders - it appears that yanbf.api.hansol.ca:443 is currently refusing connections, which is causing the generation process to fail. I've attached a log file showing the exceptions that are generated,

```
Tue Nov 29 2022 00:37:24 GMT-0500 (Eastern Standard Time): generator.py:62: DeprecationWarning: LINEAR is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BILINEAR instead.

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time): Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connection.py", line 174, in _new_conn

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connectionpool.py", line 787, in urlopen
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\util\retry.py", line 592, in increment
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='yanbf.api.hansol.ca', port=443): Max retries exceeded with url: /banner/IRBO (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EE97F05FF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\Lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 120, in run

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connection.py", line 358, in connect
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000001EE97F05FF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\requests\adapters.py", line 489, in send

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\urllib3\connectionpool.py", line 1042, in _validate_conn

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\Lib\site-packages\cx_Freeze\initscripts\Console.py", line 16, in run
  File "generator.py", line 389, in <module>
  File "generator.py", line 338, in start

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\requests\sessions.py", line 587, in request
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\requests\sessions.py", line 701, in send

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "generator.py", line 188, in downloadfromapi
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\requests\api.py", line 73, in get
  File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\requests\api.py", line 59, in request

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time): Using ROM path: C:\Users\danla\Downloads
Output file: Pokemon Black (U).nds.cia
Getting gamecode...
Extracting and resizing icon...
Getting ROM titles...
Creating SMDH...
Checking API if a custom banner or sound is provided...

Tue Nov 29 2022 00:37:27 GMT-0500 (Eastern Standard Time):   File "C:\hostedtoolcache\windows\Python\3.10.7\x64\lib\site-packages\requests\adapters.py", line 565, in send
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='yanbf.api.hansol.ca', port=443): Max retries exceeded with url: /banner/IRBO (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EE97F05FF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
```

I also confirmed that I'm able to ping the address but am not able to get a response on port 443.

```
Test-NetConnection -ComputerName yanbf.api.hansol.ca -Port 443
WARNING: TCP connect to (192.18.143.187 : 443) failed

ComputerName           : yanbf.api.hansol.ca
RemoteAddress          : 192.18.143.187
RemotePort             : 443
InterfaceAlias         : Wi-Fi
SourceAddress          : 192.168.0.101
PingSucceeded          : True
PingReplyDetails (RTT) : 82 ms
TcpTestSucceeded       : False
```

Any suggestions on a potential fix or workaround for this issue? If there's any additional information I can provide, please let me know.

Thanks.

**Comment by lifehackerhansol** [comment]
This is fixed in version 1.6.3 that was released just about 15 minutes ago. (This website was removed as a result.)

**Comment by DanLabute** [comment]
Thank you!
